ui/src/app/views/workspace-list/component.css

```diff
@@ -108,6 +108,34 @@
   color: #216FB4;
 }
 
+.card .owner-permission {
```

Comment:
is it possible to abstract the permission box into its own class that inherits from `.card`, and then have each permission type inherit from from that?  (`.card .permission-box .owner-permission` or something)  Not sure how to deal with grandparents; new to CSS and frontend coding in general

Reply:
There's not really inheritance per-se in vanilla CSS; the typical thing to do is apply multiple CSS classes to an element as needed. When there are conflicts there are rules for which rule wins, but in general more specific selectors / later in the file wins.

So in this case, I'd pull out all these common attributes onto a `.permission` class, then anything specific to each access level I'd put in separate rules:

```
.permission {
 ...
}

.owner-permission {
  background-color: #4996A2;
}

.reader-permission {
...
```